package.json

```diff
@@ -56,56 +52,68 @@
     "geostyler-wfs-parser": "0.7.3",
     "lodash": "4.17.11",
     "moment": "2.22.2",
-    "ol": "4.6.5",
-    "react": "16.6.0",
     "react-codemirror2": "5.1.0",
     "react-color": "2.14.1",
-    "react-dom": "16.6.0",
-    "react-rnd": "9.0.3",
-    "react-scripts-ts": "3.1.0",
+    "react-rnd": "9.0.2",
```

Comment:
Why did you downgrade `react-rnd`?

Reply:
Rebase error. Fixing with upcoming commit.